openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -19,11 +21,11 @@ def __str__(self):
 
 class MultiSelectWithOtherField(MultiSelectField):
```

Comment:
There is some methods in this class where you didn't add docstring, Can you please check it again? 
like `to_python` but in formfield class we provided the docstring for same method .. Please add in this class as well ..